Reconstruction/RecCalorimeter/src/components/CreateCaloCells.h

```diff
@@ -76,11 +79,14 @@ class CreateCaloCells : public GaudiAlgorithm
   std::vector<std::string> m_fieldNames;
   /// Values of the fields describing the segmented volume
   std::vector<int> m_fieldValues;
-
+  /// Temporary: for use with MergeLayer tool
```

Comment:
MergeLayer is going to be replaced by RedoSegmentation once we can define segmentation with variable cell (layer) size ( ~ with new DD4hep).
This property won't be needed anymore.